examples/lighting-app/linux/main.cpp

```diff
@@ -89,7 +89,11 @@ int main(int argc, char * argv[])
         return -1;
     }
 
-    LightingMgr().Init();
+    if (LightingMgr().Init() != CHIP_NO_ERROR)
+    {
+        chip::DeviceLayer::PlatformMgr().Shutdown();
```

Comment:
consider assigning the result to a CHIP_ERROR and printing it out. Otherwise we seem to just shutdown and return -1 without actually printing out the cause, which is annoying to debug.

For what its worth, this is better than before, so adding a review checkmark. I would prefer an error printout though.

Reply:
Thanks for the advice! I made a commit to print the CHIP error message. The log will now show:
```
[1677690023.399104][2408322:2408322] CHIP:SVR: Manual pairing code: [34970112332]
[1677690023.399111][2408322:2408322] CHIP:SVR: Failed to initialize lighting manager: ../../third_party/connectedhomeip/examples/lighting-app/lighting-common/src/LightingManager.cpp:29: CHIP Error 0x0000002F: Invalid argument
[1677690023.399141][2408322:2408322] CHIP:DL: writing settings to file (/tmp/chip_counters.ini-3ZHjQa)
[1677690023.399211][2408322:2408322] CHIP:DL: renamed tmp file to file (/tmp/chip_counters.ini)
[1677690023.399219][2408322:2408322] CHIP:DL: NVS set: chip-counters/total-operational-hours = 0 (0x0)
[1677690023.399222][2408322:2408322] CHIP:DL: Inet Layer shutdown
[1677690023.399225][2408322:2408322] CHIP:DL: BLE shutdown
[1677690023.399227][2408322:2408322] CHIP:DL: System Layer shutdown
```